.github/workflows/es-int-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Elasticsearch Integration Tests
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    branches: [ develop ]
+  workflow_dispatch:
```

Comment:
Does this line need to be here if you're not specifying anything?

Reply:
I thought it would be cool to leave it in - allow people to kick off these tests manually. Since there's no inputs required to run tests, I didn't define any inputs under the `workflow_dispatch:` line

Reply:
Cool as long as folks know to do that lgtm :)